### ISSUE TITLE
Connection Refused while trying to ssh to VIC appliance during Scale Testing  

### ISSUE THREAD

**Issue by jitinkumar2018** [vmware/vic-product#2018]
Bug Report Basic Information
----------------------------

- vCenter Server version: 6.5 U2
- Embedded or external PSC: Embedded
- Filename of the OVA you deployed: vic-v1.4.3-dev-5974-b7033624 
- How was the OVA deployed? Flex Client
- Does the VIC appliance receive configuration by DHCP? No
- What stage of the Appliance Lifecycle is the VIC appliance in? Application
- Attach the VIC appliance support bundle : 
[vic_appliance_logs_2018-09-03-11-38-48.tar.gz](https://github.com/vmware/vic-product/files/2351404/vic_appliance_logs_2018-09-03-11-38-48.tar.gz)

Bug Report Detailed Information
-------------------------------

- What operation was being performed when the failure was noticed?

Performing Scale Testing from an Ubuntu VM with the following operations:
- Deployed VIC appliance via flex client
- Created 50 VCH via vic-machine-linux cli
- Created 50 containers on each VCH via VCH docker commands 
- Created 50 projects on admiral via POST apis using curl
- While adding the 50 VCH's to one of the projects in admiral, we get an error:
`curl: (7) Failed to connect to 10.197.37.192 port 9443: Connection refused`
- Also unable to ssh to the VIC appliance from the Ubuntu VM
- Tried clearing the known_hosts file in .ssh Directory but error persists
- Problem ceases to exist after 24 hours and admiral APIs work fine then

- Was adding projects by executing the POST request from another Ubuntu machine and was successful. Able to SSH to VIC appliance from this Ubuntu machine as well

**Comment by lgayatri** [comment]
@renmaosheng @martin-borisov

**Comment by renmaosheng** [comment]
Hi Lakshmi,
     do we still have the environment? are we able to log into the VIC appliance through vCenter portal? thanks.

**Comment by lgayatri** [comment]
Hi @renmaosheng , we do not have the environment as we started uptime with scale.
When the mentioned issue occurred, we were able to login to the VIC appliance from its console as well as from another VM.

**Comment by lgayatri** [comment]
```
[1218][I][2018-09-03T09:22:31.349Z][1181][HttpServletRequestImpl][breakHere][HttpServletRequestResponse]
[1219][I][2018-09-03T09:22:31.350Z][1181][HttpServletRequestImpl][breakHere][HttpServletRequestResponse]
[1220][S][2018-09-03T09:23:24.436Z][38][8282/][exceptionCaught][Listener channel exception: Connection reset by peer]
[1221][I][2018-09-03T09:23:44.614Z][29][8282/resources/epz-periodic-enumeration][doTrigger][Periodic refresh]
[1222][I][2018-09-03T09:24:31.327Z][30][8282/][lambda$schedulePeriodicCertificatesReload$1][Host https://vic-st-h2-192.eng.vmware.com:8282/: reloading all certificates]
[1223][I][2018-09-03T09:28:44.875Z][30][8282/resources/epz-periodic-enumeration][doTrigger][Periodic refresh]
[1224][I][2018-09-03T09:29:31.328Z][30][8282/][lambda$schedulePeriodicCertificatesReload$1][Host https://vic-st-h2-192.eng.vmware.com:8282/: reloading all certificates]
[1225][I][2018-09-03T09:33:22.269Z][11][8282/][performExpiredContextCacheCleanup][Cleared 0 expired tokens. took=1000]
```

**Comment by wjun** [comment]
@lgayatri Does Admiral or VIC appliance stopped responding? It looks to me port 8282 is Admiral?

**Comment by jitinkumar2018** [comment]
@wjun Admiral and VIC do not stop responding. We could SSH into VIC from a different machine. Admiral was also accessible. The only problem was the Ubuntu machine on which we ran the scale test was getting refused to connect to VIC (10.197.37.192)

**Comment by wjun** [comment]
@lgayatri Is there any possibility about network accessibility issue, since it worked again after 24 hours?

**Comment by lgayatri** [comment]
@wjun, we will try on latest build or RC2 and update. There was no network issue during the time of the test.

**Comment by jitinkumar2018** [comment]
@wjun 
We ran scale test on RC2 build. 50 VCH's, 50 containers each, 50 projects. While trying to add the 50 VCH's to one of the projects via post commands (from our ScaleTest Ubuntu VM), we again faced the error of not able to connect to the VIC appliance from our ScaleTest Ubuntu VM.

When we tried to ssh into the VIC from out ScaleTest Ubuntu VM we hit this error: 
`ssh:connect to host 10.197.37.132 port 22: Connection refused`

However, I am able to putty into the VIC from a different dev.
